### DISCUSSION TITLE
ShadCN UI CLI not respecting aliases and installing components, lib in laravel 11 project root

### DISCUSSION THREAD

**Discussion by dev-carbon** [shadcn-ui/ui#5466] [General]
I'm currently using Laravel with Breeze, Inertia.js, and React. Initially, all components installed via ShadCN were placed in `resources/js/components`. However, after running the following command to add new components:

npx shadcn@latest add [component]

The new component gets created in the root directory of my project.

Despite specifying the `--path` option, it doesn't place components under `resources/js/shadcn`, which is the intended directory.

###Steps I have tried:
- Using the `--path` option with the correct directory.
- Manually setting up aliases in `components.json` and Vite's `vite.config.js`.
- Manually moving the components and configuring aliases to point to the correct location.

I would like the ability to directly add ShadCN components to the `resources/js/shadcn` directory without having to move them manually after each addition.

Has anyone else encountered this issue, or is there a way to configure the path correctly when using `npx shadcn add`?

**Comment by discoverlance-com** [comment]
I also encountered this issue just now, I am using astrojs and I have updated my `aliases.ui` in `components.json` so I expected it to add my components to `src/components/react/ui` and it instead adds it to `src/components/ui`.

**Comment by ivan-plotnikov** [comment]
This is a bug
I’ve already created an issue.
#5433
